tests/test_validators.py

```diff
@@ -736,6 +736,29 @@ def check_something(cls, v):
     assert val_calls == [None, 'hello']
 
 
+def test_required_optional():
+    class Model(BaseModel):
+        nullable1: Optional[int] = ...
+        nullable2: Optional[int] = Field(...)
```

Comment:
Any chance you could add the following fields just to make it very explicit right here what's going on with `Any` too? Presumably that should be handled the same as `Optional` now.
```
nullable3: Any
nullable4: Any = ...
nullable5: Any = Field(...)
```
I think it could make sense to just put them in a separate test case too.

Reply:
Good idea! Done :heavy_check_mark: